iml-docker.service

```diff
@@ -13,6 +13,7 @@ WorkingDirectory=/etc/iml-docker
 ExecStartPre=-/usr/bin/docker load -i /var/lib/iml-images.tgz
 ExecStart=/usr/bin/docker stack deploy -c docker-compose.yml -c docker-compose.overrides.yml iml --resolve-image=never
 ExecStart=/bin/bash -c 'until /usr/bin/iml server list > /dev/null 2>&1; do sleep 1; done'
```

Comment:
A bit of trickyness here. We are going to need the iml command to have it's setting in place before we can actually run this check.

Reply:
I've moved copying to `ExecStart` before `iml server list`.

> If more than one command is specified, the commands are invoked sequentially in the order they appear in the unit file. If one of the commands fails (and is not prefixed with "-"), other lines are not executed, and the unit is considered failed.

https://www.freedesktop.org/software/systemd/man/systemd.service.html

So this should be fine.

Reply:
What happens if that file does not exist yet? It gets generated early on during startup.

Reply:
Also, we can't depend on the docker/volume path being static, for example it's a different path in exascaler.

Reply:
It can't not exist yet because next command will be started only after current command finishes.

I'll investigate getting the volume path.